frontend/src/components/CreateCollectionModal/components/Content/index.tsx

```diff
@@ -247,7 +248,7 @@ const Content: FC<Props> = (props) => {
 
     setIsLoading(true);
 
-    await mutateEditCollection({
+    mutateEditCollection({
```

Comment:
ah I see `await` is removed here, which means we will run the code below before the promise resolves?

```ts
setIsLoading(false);
onClose();
```

Reply:
reinstated the await